### PR TITLE
MyOnlineMeals.com

### LINKED ISSUE #336: MyOnlineMeals: a web dev project of food delivery based on the HTML and CSS
## 🚀 Feature

(A clear and concise description of what the feature is.)

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

## Motivation

The most important advantage of food delivery is by providing food whenever or wherever you want. Nowadays people are too busy at times, but food delivery service helps them to grab a bite in their busy schedule.

## Pitch

(Please explain why this feature should be implemented and how it would be used.)
1)For starting the feature 

2)After forking the repository 

3)Go to the dir named as MyOnlineMeals

4)By clicking on the index.html

5)Go to the live server and thus feature is ready to work.

### PR DESCRIPTION
**Pull request by Ayushi271**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

##Description

A food delivery website which offers services like catering, bulk ordering and ordering etc.

Fixes #336

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes the issue #336

### CODE REVIEW

**Comment by photon149** [comment]
@Ayushi271 Please mention the issue number it fixes.

**Comment by Ayushi271** [comment]
This PR is for the issue #336

**Comment by Ayushi271** [comment]
Sorry for the late reply.
There were some issues with my laptop.

**Comment by photon149** [comment]
@Ayushi271 Great Project , but mention your issue number in your Pull Request Description , where it says "Fixes #Yourissuenum".
@prakhar3062 @Kushal997-das have a look.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Ayushi271 Everything is looks good, but you have to add gif or photos about your project in your `readme.md` file.

**Comment by Ayushi271** [comment]
@photon149  I have edited it and wrote issue no. Is it okay now?

**Comment by Ayushi271** [comment]
@Kushal997-das  I have added some images of the website in the readme.md file and also fixed some broken links. Is it okay now?

File: Web Development/Basic/MyOnlineMeals/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<img src="landing page.png">
+
+##Service page
+<img src="service.png" >
+
+## Contact Page
+<img src="contact page.png" >
+
+## Food items Page
+<img src="item page.png" >
```
Comment (Kushal997-das) [inline Web Development/Basic/MyOnlineMeals/README.md:46]: @Ayushi271 Just reduce the image size.

Comment (Kushal997-das) [reply Web Development/Basic/MyOnlineMeals/README.md:46]: @Ayushi271

Comment (Ayushi271) [reply Web Development/Basic/MyOnlineMeals/README.md]: @Kushal997-das  I have reduced the size of images.

File: Web Development/Basic/MyOnlineMeals/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  ` MyOnlineMeals`
+
+  find index.html
+  and then if you use Vs Code then choose Go Live in the bottom.
+
+## Landing Page
+
+<img src="landingpage.png">
+
+##Service page
```
Comment (Kushal997-das) [inline Web Development/Basic/MyOnlineMeals/README.md:39]: Give the space after `##` .

File: Web Development/Basic/MyOnlineMeals/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+<img src="service.png" >
+
+## Contact Page
+<img src="contact page.png" >
+
+## Food items Page
+<img src="item page.png" >
+
```
Comment (Kushal997-das) [inline Web Development/Basic/MyOnlineMeals/README.md:47]: Please use this structure to reduce the image size in `readme.md` file .



You can reduce the width and height .

**Review by Kushal997-das — APPROVED** [review]
@Ayushi271 Great work !!

**Comment by Kushal997-das** [comment]
@photon149 Have a look !!

**Comment by Kushal997-das** [comment]
@prakhar3062 Have a look !

**Review by photon149 — COMMENTED** [review]
@Ayushi271 Approved from my end.

**Comment by Ayushi271** [comment]
@photon149 @Kushal997-das Thanks for guiding me and making my first contribution successful. Thanks for merging it.

**Comment by Kushal997-das** [comment]
@Ayushi271 Most welcome , feel free to take another issue :)

**Pull request merged**